src/healthsrv/server.go

```diff
@@ -12,7 +13,7 @@ const (
 )
 
 // Start starts the healthcheck server on $host:$port and blocks. It only returns if the server fails, with the indicative error
-func Start(host string, port int, minioClient minio.CloudStorageClient) error {
+func Start(host string, port int, minioClient minio.Client) error {
```

Comment:
would it be better to make this a pointer to *minio.Client so we don't have to de-reference the pointer at https://github.com/deis/minio/pull/74/files#diff-f6ee15f775b88714fe6122ff877b9e42R170?